python_sdk/pyproject.toml

```diff
@@ -308,6 +308,13 @@ max-complexity = 17
 [tool.ruff.lint.per-file-ignores]
 
 
+"infrahub_sdk/client.py" = [
+    ##################################################################################################
+    # Review and change the below later                                                              #
+    ##################################################################################################
+    "PLR0904",  # Too many public methods
+]
+
 "python_sdk/infrahub_sdk/client.py" = [
```

Comment:
is the section below also required ? or is it never gonna be used anyway ?

Reply:
It is required not that we're also running `ruff check python_sdk --config python_sdk/pyproject.toml` in the CI pipeline with that command the original paths are incorrect.

Until we've actually split the repos I think it's nice to be able to run ruff with just one command and have everything covered.

Reply:
is this command still needed with this fix ? or can we revert to using `ruff check .` at the root level ?

Reply:
Spoke to @wvandeun separately about this as the format issue came up elsewhere. I didn't understand why you needed the original change but I think now that it was due to issues with `invoke format` or `invoke sdk.format` is that correct? I think the main problem here might be that the invoke format commands are setup incorrectly in that they target the SDK pyproject.toml files so the paths gets mixed up. Would that be in line with what you saw?